Makefile

```diff
@@ -15,9 +15,9 @@ intel:
 	ln -s ./config/config.LINUX_INTEL.mk common.mk
 	( cd src && make) || exit 1; 
 	( cd src_cs && make) || exit 1;
-	f2py  --fcompiler=intel --f90flags=-r8 -c -m pyxlight src/pyxlight.pyf src/libxfoil.a
-	f2py  --fcompiler=intel --f90flags=-r8 -c -m pyxlight_cs src_cs/pyxlight_cs.pyf src_cs/libxfoil_cs.a
-	mv *.so ./python
+	f2py  --fcompiler=intel --f90flags=-r8 -c -m libpyxlight src/libpyxlight.pyf src/libxfoil.a
```

Comment:
For consistency we probably want to call this `libxlight` instead

Reply:
Good point, I'll set it to that.

Reply:
Got it. I'll update that.